gemd/entity/object/material_spec.py

```diff
@@ -58,7 +61,25 @@ def properties(self):
 
     @properties.setter
     def properties(self, properties):
-        self._properties = validate_list(properties, PropertyAndConditions)
+        def _template_check(x: PropertyAndConditions) -> PropertyAndConditions:
```

Comment:
Same thing. This is identical to `conditions` and `parameters`, and could probably be re-used.

Reply:
Addressed as well.